fac29/hoda-e-commerce-backend
# Create session table

How to test: 

1. Create a session ```createSession(1)```
2. Check that the session has been created in the sessions table of the database 
3. Look for the session id 
4. Use ```getSession(sid: Session['sid'])```
